presto-main/src/main/java/com/facebook/presto/sql/planner/GroupingOperationRewriter.java

```diff
@@ -57,16 +54,9 @@ public GroupingOperationRewriter(QuerySpecification queryNode, Analysis analysis
         this.groupIdSymbol = requireNonNull(groupIdSymbol, "groupIdSymbol is null");
     }
 
-    @Override
-    public Expression rewriteExpression(Expression node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
+    public Expression rewrite(GroupingOperation expression)
```

Comment:
this is a utility class now, make it final.

Rename this method to `rewriteGroupingOperation` and make it static